users/dierkes/preprocessing/segment_cutting.py

```diff
@@ -0,0 +1,85 @@
+import soundfile as sf
+import numpy as np
+from i6_core.lib import corpus
+import multiprocessing
+import tqdm
+
+from sisyphus import Job, Task, tk
```

Comment:
```suggestion
import multiprocessing
import numpy as np
import soundfile as sf
import tqdm

from sisyphus import Job, Task, tk

from i6_core.lib import corpus
```
See https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTING.md